pde_superresolution/weno_test.py

```diff
@@ -28,21 +28,24 @@
 class WENOTest(parameterized.TestCase):
 
   def test_calculate_omega_smooth(self):
-    u = np.zeros(5)
+    u = np.linspace(0, 1)
```

Comment:
can you explicitly add the optional `num` parameter, just so it's clear what's going on?